mujoco_interface_node:
#   ros__parameters:
  camera_focal_point: [0.0, 0.0, 0.5] # [0, 0, 0.5]
  camera_distance: 1.5 #2.5
  camera_azimuth: -90 #135.0
  camera_elevation: 5 #-30.0
  camera_orthographic: true
